### PR TITLE
feat: Reorganizing access rights

### PR DESCRIPTION
**Pull request by mtrunkat**
I slightly reorganized access rights sections as access rights should not be above the organization account - those are 2 different features, so now they are both under the "Collaboration":

obrazek

### CODE REVIEW

**Comment by B4nan** [comment]
I guess we should enable the PR checks for all branches, now they are used only for PRs targetting master (and I don't think we did that on purpose).

**Comment by mtrunkat** [comment]
@B4nan let's do it. It's a good idea. I am sometimes creating these PR chains for an easier review

#532 ↩️ #542 ↩️ #543

**Comment by B4nan** [comment]
Here you go: 

https://github.com/apify/apify-docs/commit/508d5f265f01bd13d7638c913cf16c7857105182

**Review by PerVillalva — APPROVED** [review]
Just capitalized a few instances of the word Actor to align with the new guidelines. Otherwise, everything seems to be fine.

File: sources/platform/actors/running/runs_and_builds.md [hunk 1]
```diff
@@ -112,5 +112,5 @@ All **actor runs** are deleted along with their default storages (key-value stor
 
 ## Sharing
 
-You can share your actor runs with other Apify users via the [access rights](../../access_rights/index.md) system.
+You can share your actor runs with other Apify users via the [access rights](../../collaboration/index.md) system.
```
Comment (TheoVasilis) [inline sources/platform/actors/running/runs_and_builds.md:115]:
```suggestion
You can share your Actor runs with other Apify users via the [access rights](../../collaboration/index.md) system.
```

File: sources/platform/actors/running/tasks.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ All your tasks are listed at [https://console.apify.com/actors/tasks](https://co
 
 You can create tasks both for your personal actors and for those made by someone else. Just [search Apify Store](https://console.apify.com/store) to find an actor you want to use and click the **Create new task** button.
 
-Like any other resource, you can share your actor tasks with other Apify users via the [access rights](../../access_rights/index.md) system.
+Like any other resource, you can share your actor tasks with other Apify users via the [access rights](../../collaboration/index.md) system.
```
Comment (TheoVasilis) [inline sources/platform/actors/running/tasks.md:20]:
```suggestion
Like any other resource, you can share your Actor tasks with other Apify users via the [access rights](../../collaboration/index.md) system.
```

File: sources/platform/actors/running/tasks.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ To make a task easier to identify, you can give it a name, title, and descriptio
 
 ### Settings
 
-Under the **Settings** tab of their detail page, you can grant [access rights](../../access_rights/index.md) to other Apify users.
+Under the **Settings** tab of their detail page, you can grant [access rights](../../collaboration/index.md) to other Apify users.
```
Comment (TheoVasilis) [inline sources/platform/actors/running/tasks.md:44]:
```suggestion
You can grant [access rights](../../collaboration/index.md) to other Apify users under the **Settings** tab of their detail page.
```

File: sources/platform/collaboration/access_rights.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+sidebar_position: 12
+category: platform
+slug: /collaboration/access-rights
+---
+
+**Manage permissions for your private resources such as Actors, Actor runs, and storages. Allow other users to read, run, modify, or build new versions.**
+
+---
+
+You can easily and securely share your own resources - Actors, tasks, key-value stores, datasets, and request queues with other users using a [granular](https://www.google.com/search?client=firefox-b-d&q=define+granular+permissions) permissions system. This enables you to, for example, allow your colleague to run an [Actor](../actors/index.md) or view a [dataset](../storage/dataset.md) but not modify them.
```
Comment (TheoVasilis) [inline sources/platform/collaboration/access_rights.md:13]:
```suggestion
You can easily and securely share your own resources - Actors, tasks, key-value stores, datasets, and request queues - with other users by using a [granular](https://www.google.com/search?client=firefox-b-d&q=define+granular+permissions) permissions system. This enables you, for example, to let your colleague run an [Actor](../actors/index.md) or view a [dataset](../storage/dataset.md) but not modify it.
```

File: sources/platform/collaboration/access_rights.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+> To be able to grant access rights to another user, you must have a **username** set in [account settings](https://console.apify.com/account?tab=settings).
+
+You can also grant permission to update an Actor and build a new version.
+[Storages](../storage/index.md) (key-value stores, request queues, and datasets) are sharable the same way with, for example, just **read** permission or a combination of both **read** and **write** permissions.
```
Comment (TheoVasilis) [inline sources/platform/collaboration/access_rights.md:18]:
```suggestion
[Storages](../storage/index.md) (key-value stores, request queues, and datasets) are sharable in the same way as a **read** permission or a combination of both **read** and **write** permissions.
```

File: sources/platform/collaboration/access_rights.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+To share an Actor, task, key-value store, request queue, or dataset, click on the **Actions** button in the top-right corner and select **Share**. You can add a collaborator using their **user ID**, **email**, or **username**. Once you have added the collaborator, you can configure their permissions.
```
Comment (TheoVasilis) [inline sources/platform/collaboration/access_rights.md:20]:
```suggestion
To share an Actor, task, key-value store, request queue, or dataset, click on the **Actions** button in the top-right corner and select **Share**. You can add a collaborator by using their **user ID**, **email**, or **username**. Once you have added the collaborator, you can configure their permissions.
```

File: sources/platform/collaboration/index.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+        <td><strong><a href="/platform/collaboration/access-rights">Access rights</a></strong></td>
+        <td>Enables you to grant access to another user for a certain resource you own. This way, you can share results with your client, or two engineers can collaborate on developing one Actor.</td>
+    </tr>
+    <tr>
+        <td><strong><a href="/platform/collaboration/organization-account">Organization account</a></strong></td>
+        <td>Apify's organization account allows multiple engineers to collaborate on team projects with role-specific access permissions.</td>
+    </tr>
+    <tr>
+        <td><strong><a href="/platform/actors/publishing">Publishing in Apify Store</a></strong></td>
+        <td>Another way to share your Actor with other users is to publish it in <a href="https://apify.com/store">Apify Store</a>. When publishing your Actor, you can make it a Paid Actor and get paid by the users benefiting from your tool. For more, read the <a href="/platform/actors/publishing">publishing and monetization</a> section.</td>
```
Comment (TheoVasilis) [inline sources/platform/collaboration/index.md:26]:
```suggestion
        <td>Another way to share your Actor with other users is to publish it in <a href="https://apify.com/store">Apify Store</a>. When publishing your Actor, you can make it a Paid Actor and get paid by the users benefiting from your tool. For more information, read the <a href="/platform/actors/publishing">publishing and monetization</a> section.</td>
```

File: sources/platform/collaboration/organization_account/index.md [hunk 8]
```diff
@@ -62,7 +62,7 @@ Actor and task runs are billed to the account they are started from. **Always ma
 
 The Free and Personal plans are [restricted to 9 seats](https://apify.com/pricing), while the Team plan comes with 9, which you can increase later. If you are on the Team plan and want to add some more seats to the 9 it comes with, head over to your [Billing page and click on Subscriptions](https://console.apify.com/billing-new#/subscription). Click on **Upgrade**, and, in the **Add-ons** section, and add as many additional seats as you need. Then, just complete the upgrade process, and you're done.
```
Comment (TheoVasilis) [inline sources/platform/collaboration/organization_account/index.md:63]:
```suggestion
The Free and Personal plans are [restricted to 9 seats](https://apify.com/pricing), while the Team plan comes with a minimum of 9, which you can increase later. If you are on the Team plan and want to add some more seats to the 9 it comes with, head over to your [Billing page and click on Subscriptions](https://console.apify.com/billing-new#/subscription). Click on **Upgrade**, and, in the **Add-ons** section, and add as many additional seats as you need. Then, just complete the upgrade process, and you're done.
```

File: sources/platform/schedules.md [hunk 9]
```diff
@@ -151,4 +151,4 @@ Additionally, you can use the following shortcut expressions:
 
 ## Sharing
 
-You can invite other Apify users to view or modify your schedules using the [access rights](./access_rights/index.md) system. See the [full list of permissions](./access_rights/list_of_permissions.md).
+You can invite other Apify users to view or modify your schedules using the [access rights](./collaboration/index.md) system. See the [full list of permissions](./collaboration/list_of_permissions.md).
```
Comment (TheoVasilis) [inline sources/platform/schedules.md:154]:
```suggestion
You can invite other Apify users to view or modify your schedules with the [access rights](./collaboration/index.md) system. See the [full list of permissions](./collaboration/list_of_permissions.md).
```

File: sources/platform/storage/dataset.md [hunk 10]
```diff
@@ -321,7 +321,7 @@ By default, the whole result is wrapped in an `<items/>` element, while each pag
 
 ## Sharing {#sharing}
 
-You can invite other Apify users to view or modify your datasets using the [access rights](../access_rights/index.md) system. See the [full list of permissions](../access_rights/list_of_permissions.md).
+You can invite other Apify users to view or modify your datasets using the [access rights](../collaboration/index.md) system. See the [full list of permissions](../collaboration/list_of_permissions.md).
```
Comment (TheoVasilis) [inline sources/platform/storage/dataset.md:324]:
```suggestion
You can invite other Apify users to view or modify your datasets with the [access rights](../collaboration/index.md) system. See the [full list of permissions](../collaboration/list_of_permissions.md).
```

File: sources/platform/storage/index.md [hunk 11]
```diff
@@ -191,7 +191,7 @@ For example, the storage names **janedoe~my-storage-1** and **janedoe~web-scrape
 
 ## Sharing {#sharing}
 
-You can invite other Apify users to view or modify your storages using the [access rights](../access_rights/index.md) system. See the [full list of permissions](../access_rights/list_of_permissions.md).
+You can invite other Apify users to view or modify your storages using the [access rights](../collaboration/index.md) system. See the [full list of permissions](../collaboration/list_of_permissions.md).
```
Comment (TheoVasilis) [inline sources/platform/storage/index.md:194]:
```suggestion
You can invite other Apify users to view or modify your storages with the [access rights](../collaboration/index.md) system. See the [full list of permissions](../collaboration/list_of_permissions.md).
```

File: sources/platform/storage/key_value_store.md [hunk 12]
```diff
@@ -204,7 +204,7 @@ You can compress a record and use the [Content-Encoding request header](https://
 
 ## Sharing
 
-You can invite other Apify users to view or modify your key-value stores using the [access rights](../access_rights/index.md) system. See the [full list of permissions](../access_rights/list_of_permissions.md).
+You can invite other Apify users to view or modify your key-value stores using the [access rights](../collaboration/index.md) system. See the [full list of permissions](../collaboration/list_of_permissions.md).
```
Comment (TheoVasilis) [inline sources/platform/storage/key_value_store.md:207]:
```suggestion
You can invite other Apify users to view or modify your key-value stores with the [access rights](../collaboration/index.md) system. See the [full list of permissions](../collaboration/list_of_permissions.md).
```

File: sources/platform/storage/request_queue.md [hunk 13]
```diff
@@ -217,7 +217,7 @@ See the [API documentation](/api/v2#/reference/request-queues) for a detailed br
 
 ## Sharing {#sharing}
 
-You can invite other Apify users to view or modify your request queues using the [access rights](../access_rights/index.md) system. See the [full list of permissions](../access_rights/list_of_permissions.md).
+You can invite other Apify users to view or modify your request queues using the [access rights](../collaboration/index.md) system. See the [full list of permissions](../collaboration/list_of_permissions.md).
```
Comment (TheoVasilis) [inline sources/platform/storage/request_queue.md:220]:
```suggestion
You can invite other Apify users to view or modify your request queues with the [access rights](../collaboration/index.md) system. See the [full list of permissions](../collaboration/list_of_permissions.md).
```

**Review by TheoVasilis — CHANGES_REQUESTED** [review]

**Pull request merged**